ozaria/site/components/play/PageUnitMap/common/UnitMapBackground.vue

```diff
@@ -148,15 +158,25 @@ export default Vue.extend({
       v-if="showNavDots"
       id="dot-nav"
     >
+      <div
+        class="arrow left"
+        :class="{ inactive: campaignPage === 1 }"
+        @click="previousPage"
+      />
       <ul>
         <li
           v-for="page in totalPages"
           :key="page"
-          :class="{ active: page === campaignPage }"
+          :class="{ selected: page === campaignPage }"
```

Comment:
not blocking but why do we update the class name ?

Reply:
@smallst For me `active` suggested that the button is active, and should be clickable. 
But this button should be in non-clickable (inactive) state, because it is already selected.